environments/dev/manifests/thanos-store-shard0-statefulSet.yaml

```diff
@@ -37,6 +37,20 @@ spec:
         - --grpc-address=0.0.0.0:10901
         - --http-address=0.0.0.0:10902
         - --objstore.config=$(OBJSTORE_CONFIG)
+        - --experimental.enable-index-cache-postings-compression
+        - |-
+          --index-cache.config="config":
+            "addresses":
```

Comment:
How we ensure how many items we can cache? :thinking:

Reply:
@kakkoyun ?

Reply:
We limit the max item size and max memory, this is how we control maximum number of items in the cache. For our own deployment, we will have another internal PR I'll address them over there.

Also I have created an issue as a follow up to this https://github.com/observatorium/configuration/issues/243